tests/lib_files_test.py

```diff
@@ -240,3 +235,37 @@ def test_warpvnd_import(fc):
             simulationType=fc.sr_sim_type,
         ),
     )
+
+
+def _get_file(fc, api_name):
+    from pykern import pkunit, pkcompat
+    from sirepo import sim_data
+    from pykern.pkcollections import PKDict
+    from pykern.pkunit import pkre, pkeq
+
+    d = fc.sr_sim_data("Sample from Image")
+    s = sim_data.get_class(fc.sr_sim_type)
+    r = fc.sr_get(
+        api_name,
+        params=PKDict(
+            simulation_type=fc.sr_sim_type,
+            filename="sample.tif",
+        ),
+        data=PKDict(),
+        redirect=False,
+    )
+    pkre("/tif", r.mimetype)
+    f = s.lib_file_resource_path("sample.tif")
```

Comment:
temporary variable not needed

Reply:
[b90549d](https://github.com/radiasoft/sirepo/pull/6348/commits/b90549d0140828850c8128228af2c1f76a0b1ffa)